ush/python/pygfs/task/analysis.py

```diff
@@ -77,9 +102,9 @@ def get_obs_dict(self) -> Dict[str, Any]:
     def get_bias_dict(self) -> Dict[str, Any]:
         """Compile a dictionary of observation files to copy
 
-        This method uses the OBS_LIST configuration variable to generate a dictionary
-        from a list of YAML files that specify what observation bias correction files
-        are to be copied to the run directory from the observation input directory
+        This method extracts 'observers' from the JEDI yaml and from that list, extracts a list of
+        observation bias correction files that are to be copied to the run directory
+        from the observation input directory
```

Comment:
```suggestion
        from the component directory.
        TODO: COM_ATMOS_ANALYSIS_PREV is hardwired here and this method is not appropriate in `analysis.py` and should be implemented in the component where this is applicable.
```
@RussTreadon-NOAA We should implement this in `atm_analysis.py`

Reply:
GDAS (and GFS) ATM and EnKF analysis use radiance (and later aircraft) bias correction files from the GDAS.  Do we replicate `get_bias_dict` `atm_analysis.py` and `atmens_analysis.py`?  Other JEDI DA modes (land, ocean) may also use GDAS bias correction files in the future.  Where's the best place to put a staging method which copies files from the GDAS to various JEDI DA modes?